tools/rules_config/_config.sh

```diff
@@ -19,14 +19,11 @@ SONAR_URL=http://localhost:9000
 # SONAR_URL=
 
 # new tag to add to rules
-# TAG_ECOCONCEPTION=eco-design
-# TAG_ECOCONCEPTION=eco-sql
+TAG_ECOCONCEPTION=eco-design
 # TAG_ECOCONCEPTION=eco-design-test
-TAG_ECOCONCEPTION=test1
-# TAG_ECOCONCEPTION=
 
 # list of rule keys that will be updated with new tag
-RULES_KEYS=css:S4655,php:S2014,Web:ItemTagNotWithinContainerTagCheck
+RULES_KEYS=css:S4655,php:S2014,Web:ItemTagNotWithinContainerTagCheck,java:S1696
```

Comment:
please keep only your rule `java:S1696` (other rules are only for testing the system